EntityLayer/Entities/Asset.cs

```diff
@@ -15,5 +16,10 @@ public class Asset:IAsset
         public DateTime LastSync { get; set; }
         public int FloorMapId { get; set; }
         public bool Active { get; set; }
+
+        public virtual FloorMap FloorMap { get; set; }
+        //public virtual ICollection<AssetPositionHistory> AssetPositionHistories { get; set; } = new List<AssetPositionHistory>();
```

Comment:
comment?

Reply:
This code will be used after creating the AssetZoneHistory class.